site/app/templates/polls/ViewPoll.twig

```diff
@@ -63,10 +63,13 @@
                                 "content" : option.getResponse(),
                                 "style": "display: inline"
                             } only %}
+                            {% if option.isCorrect() and poll.isReleaseAnswer()%}
+                                <p id='correctTag' style="color: #0eed0e; margin-left: 0rem ">correct!</p>
```

Comment:
As per the [style guide](https://submitty.org/developer/software_and_system_design/coding_style_guide/css), this ID should be dash-separated.  e.g., `correct-tag` instead of `correctTag`.

Reply:
Done